newsroom/wire/views.py

```diff
@@ -161,6 +193,7 @@ def get_home_data():
         "topics": topics,
         "ui_config": get_resource_service("ui_config").get_section_config("home"),
         "groups": app.config.get("WIRE_GROUPS", []),
+        "personalizedDashboards": get_personal_dashboards_data(user, company, topics),
```

Comment:
Loading the items on initial load goes against the responsive changes we did previously. At the moment we leave it to the client to request these items, and show a loading spinner until the items arrive on the client.

Reply:
We are currently doing that loading in the dashboard though.

And I agree about caching, similar to what we have with current dashboard items